core/shared/src/main/scala/matryoshka/instances/fixedpoint/package.scala

```diff
@@ -228,7 +229,7 @@ package object fixedpoint {
     }
 
     // TODO: Would be nice to have this in ApplicativeOps
-    def almostEqual[F[_]: Applicative, A: Equal](a: F[A], b: F[A]): F[Boolean] =
+    def almostEqual[F[_]: Applicative, B: Equal](a: F[B], b: F[B]): F[Boolean] =
```

Comment:
It’s only in there because it exists as a helper for `≈`, but yeah, it could be somewhere more general.